recipe/meta.yaml

```diff
@@ -7,12 +7,12 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  url: {{ cran_mirror }}/src/contrib/Archive/dggridR/dggridR_{{ version }}.tar.gz
-  sha256: 83a9f2ea7c9b4152baa6f47da72b72d73835ed45e62b71bcf6deccebc63cc086
+  url: https://github.com/r-barnes/dggridR/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Looks like [upstream is actively planning to get back on CRAN](https://github.com/r-barnes/dggridR/issues/42). We should keep an eye out for this and be sure to switch back to the CRAN upstream rather than GitHub.

Reply:
Yes totally I will make the changes for when the package will be on CRAN !